scripts/kmod_build.py

```diff
@@ -130,7 +136,7 @@ def build_module(output_dir, module_ko, kdir, module_dir, make_args, extra_cflag
     # Don't abspath cross_compile if it's just a prefix for something already
     # inside $PATH (i.e. it doesn't contain a directory part):
     cross_compile = args.cross_compile
-    if os.path.dirname(args.cross_compile):
+    if cross_compile and os.path.dirname(args.cross_compile):
```

Comment:
We need the same logic for args.cc and args.hostcc

The caller could specify a relative path to the compiler to use, so kmod_build should make it absolute

Reply:
Done. I've introduced small function to avoid code duplication